CMakeLists.txt

```diff
@@ -34,6 +34,33 @@ if (MSVC)
   set(CMAKE_CXX_STANDARD_REQUIRED ON)
 endif()
 
+set(COMPILER ${CMAKE_CXX_COMPILER_ID})
+if (COMPILER STREQUAL "GNU")
+  set(COMPILER "GCC") # perfer printing GCC instead of GNU
```

Comment:
Note that there is [this (now outdated) GCC check deeper in this CMakeLists](https://github.com/dolphin-emu/dolphin/blob/68d439a4b3b37bd1a0478f473213e0bdb8331e35/CMakeLists.txt#L282-L285), removing that probably should go with this PR.